### ISSUE TITLE
LBAS does not check for necessary changes when checkFatigue is disabled

### ISSUE THREAD

**Issue by CripsyBekon** [perryhuynh/kcauto#292]
#### Environment details

* kcauto version (version on first line of `CHANGELOG.md`):6.8.3
* Viewer/browser:Chromium
* Operating system and/or VM:Linux

#### Issue variability
Is the issue consistent (can be reproduced consistently) or is it intermittent (only happens sometimes)?:
Consistent

#### Issue

When Fatigue check is disabled, the LBAS does not check if it is required to change modes/ignoring necessary mode changes (defense,combat)specified in the config. It will check for a resupply if necessary(indicated in the second log), but it will not change the modes even if its wrong.

![LBAS](https://user-images.githubusercontent.com/32804784/45928834-2f520b00-bf8d-11e8-8bed-a9a3434c5075.png)

This was my lbas base after resupplying. 

Expected Outcome: Mode switched to defense after resupply, then switched to the second fleet
Actual Outcome: script switched to the second fleet after resupply

#### Log
```
[log] CLICK on L(288,411)@S(0)[0,0 1918x1021] (523 msec)
[log] CLICK on L(273,352)@S(0)[0,0 1918x1021] (524 msec)
[log] CLICK on L(1137,755)@S(0)[0,0 1918x1021] (527 msec)
[2018-09-23 23:38:45] Begin resupplying LBAS groups.
[log] CLICK on L(364,673)@S(0)[0,0 1918x1021] (521 msec)
[2018-09-23 23:38:51] Checking LBAS group #1 state.
[2018-09-23 23:38:53] Checking LBAS group #2 state.
[log] CLICK on L(1046,263)@S(0)[0,0 1918x1021] (523 msec)
[2018-09-23 23:38:57] Checking LBAS group #3 state.
[log] CLICK on L(1147,263)@S(0)[0,0 1918x1021] (521 msec)
```
```
[2018-09-23 23:57:45] At home
[2018-09-23 23:57:45] Navigating to combat screen.
[log] CLICK on L(295,438)@S(0)[0,0 1918x1021] (525 msec)
[log] CLICK on L(417,456)@S(0)[0,0 1918x1021] (521 msec)
[log] CLICK on L(1118,754)@S(0)[0,0 1918x1021] (522 msec)
[2018-09-23 23:57:52] Begin resupplying LBAS groups.
[log] CLICK on L(418,663)@S(0)[0,0 1918x1021] (525 msec)
[2018-09-23 23:57:57] Checking LBAS group #1 state.
[log] CLICK on L(1187,400)@S(0)[0,0 1918x1021] (524 msec)
[2018-09-23 23:57:58] Resupplying LBAS group #1.
[2018-09-23 23:58:03] Checking LBAS group #2 state.
[log] CLICK on L(1046,263)@S(0)[0,0 1918x1021] (521 msec)
[2018-09-23 23:58:06] Checking LBAS group #3 state.
[log] CLICK on L(1147,263)@S(0)[0,0 1918x1021] (522 msec)
[log] CLICK on L(1166,398)@S(0)[0,0 1918x1021] (525 msec)
[2018-09-23 23:58:08] Resupplying LBAS group #3.
```
#### Config
```
LBASGroup1Nodes = 
LBASGroup2Nodes = T, T
LBASGroup3Nodes = Z2, Z2
LBASGroups = 1, 2, 3
Map = E-5
MiscOptions = 
```

**Comment by mrmin123** [comment]
This behavior was actually by design, putting the onus of proper fleet and
LBAS setup on the user. I can see why this is a bit confusing now, since
future updates allowed kcauto to manage LBAS states when check fatigue is
turned on.

I will consider putting in LBAS state management even when check fatigue is
not enabled. For the time being, please ensure that your fleet and LBAS are
set correctly before launching kcauto.

**Comment by CripsyBekon** [comment]
To be honest, this bug report was more for consistency purposes than anything, because it is very edge case, or even impossible for people to sortie out to events or world 6 without fatigue check enabled, given the nature of events and taiha's. 

I was able to produce this only because I was testing something for a friend, and I din't have fatigue check enabled on that config. Since it is working as intended, it is probably fine to keep it as it is.

**Comment by mrmin123** [comment]
I do agree for consistency it should manage LBAS states even when Fatigue Check is turned off. I just want to double check that there won't be any unintended side effects with how kcauto would assume the LBAS states.

**Comment by mrmin123** [comment]
I believe this is resolved as of [v8.0.0-rc1](https://github.com/mrmin123/kcauto/releases/tag/v8.0.0-rc1).
